<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>showLoan</class>
    <widget class="QDialog" name="showLoan">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>683</width>
                <height>515</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Dialog</string>
        </property>
        <widget class="QLabel" name="logoLabel">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>631</width>
                    <height>71</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>A Charming Font Expanded</family>
                    <pointsize>36</pointsize>
                </font>
            </property>
            <property name="text">
                <string>L-Book</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
            <property name="textInteractionFlags">
                <set>Qt::NoTextInteraction</set>
            </property>
        </widget>
        <widget class="QGroupBox" name="groupBox_3">
            <property name="geometry">
                <rect>
                    <x>50</x>
                    <y>330</y>
                    <width>591</width>
                    <height>81</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="title">
                <string>Information:</string>
            </property>
            <widget class="QDateEdit" name="startDate">
                <property name="geometry">
                    <rect>
                        <x>139</x>
                        <y>40</y>
                        <width>154</width>
                        <height>21</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLabel" name="label_19">
                <property name="geometry">
                    <rect>
                        <x>20</x>
                        <y>40</y>
                        <width>83</width>
                        <height>21</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Start Date:</string>
                </property>
            </widget>
            <widget class="QLabel" name="endLabel">
                <property name="geometry">
                    <rect>
                        <x>299</x>
                        <y>40</y>
                        <width>75</width>
                        <height>21</height>
                    </rect>
                </property>
                <property name="text">
                    <string>DeadLine:</string>
                </property>
            </widget>
            <widget class="QDateEdit" name="endDate">
                <property name="geometry">
                    <rect>
                        <x>407</x>
                        <y>40</y>
                        <width>154</width>
                        <height>21</height>
                    </rect>
                </property>
            </widget>
        </widget>
        <widget class="QWidget" name="layoutWidget_1">
            <property name="geometry">
                <rect>
                    <x>340</x>
                    <y>480</y>
                    <width>335</width>
                    <height>32</height>
                </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                    <widget class="QPushButton" name="finishedButton">
                        <property name="font">
                            <font>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="text">
                            <string>Mark as Finished</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QPushButton" name="deleteButton">
                        <property name="font">
                            <font>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="text">
                            <string>Delete</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QPushButton" name="exitButton">
                        <property name="font">
                            <font>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="text">
                            <string>Exit</string>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget_2">
            <property name="geometry">
                <rect>
                    <x>50</x>
                    <y>420</y>
                    <width>591</width>
                    <height>32</height>
                </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                    <widget class="QLabel" name="dayEndLabel">
                        <property name="maximumSize">
                            <size>
                                <width>120</width>
                                <height>16777215</height>
                            </size>
                        </property>
                        <property name="font">
                            <font>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="text">
                            <string>Day to End:</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QLabel" name="endDayLabel">
                        <property name="font">
                            <font>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="text">
                            <string/>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QLabel" name="extendLabel">
                        <property name="font">
                            <font>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="text">
                            <string>Extend loan duration:</string>
                        </property>
                        <property name="textInteractionFlags">
                            <set>Qt::NoTextInteraction</set>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QLineEdit" name="extendLine">
                        <property name="maximumSize">
                            <size>
                                <width>50</width>
                                <height>16777215</height>
                            </size>
                        </property>
                        <property name="font">
                            <font>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QPushButton" name="extendButton">
                        <property name="maximumSize">
                            <size>
                                <width>100</width>
                                <height>16777215</height>
                            </size>
                        </property>
                        <property name="font">
                            <font>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="text">
                            <string>Extend</string>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QGroupBox" name="groupBox">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>70</y>
                    <width>328</width>
                    <height>251</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="title">
                <string>Account:</string>
            </property>
            <widget class="QWidget" name="layoutWidget_3">
                <property name="geometry">
                    <rect>
                        <x>11</x>
                        <y>40</y>
                        <width>281</width>
                        <height>191</height>
                    </rect>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                        <layout class="QVBoxLayout" name="verticalLayout_3">
                            <item>
                                <widget class="QLabel" name="label_2">
                                    <property name="font">
                                        <font>
                                            <pointsize>10</pointsize>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string>Username:</string>
                                    </property>
                                    <property name="textInteractionFlags">
                                        <set>Qt::NoTextInteraction</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="label_3">
                                    <property name="font">
                                        <font>
                                            <pointsize>10</pointsize>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string>Name:</string>
                                    </property>
                                    <property name="textInteractionFlags">
                                        <set>Qt::NoTextInteraction</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="label_4">
                                    <property name="font">
                                        <font>
                                            <pointsize>10</pointsize>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string>Surname:</string>
                                    </property>
                                    <property name="textInteractionFlags">
                                        <set>Qt::NoTextInteraction</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="label_5">
                                    <property name="font">
                                        <font>
                                            <pointsize>10</pointsize>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string>Class:</string>
                                    </property>
                                    <property name="textInteractionFlags">
                                        <set>Qt::NoTextInteraction</set>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QVBoxLayout" name="verticalLayout_4">
                            <item>
                                <widget class="QLabel" name="usernameLabel">
                                    <property name="font">
                                        <font>
                                            <pointsize>10</pointsize>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                    <property name="textInteractionFlags">
                                        <set>Qt::NoTextInteraction</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="nameLabel">
                                    <property name="font">
                                        <font>
                                            <pointsize>10</pointsize>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                    <property name="textInteractionFlags">
                                        <set>Qt::NoTextInteraction</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="surnameLabel">
                                    <property name="font">
                                        <font>
                                            <pointsize>10</pointsize>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                    <property name="textInteractionFlags">
                                        <set>Qt::NoTextInteraction</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="classLabel">
                                    <property name="font">
                                        <font>
                                            <pointsize>10</pointsize>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                    <property name="textInteractionFlags">
                                        <set>Qt::NoTextInteraction</set>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                </layout>
            </widget>
        </widget>
        <widget class="QGroupBox" name="publicationBox">
            <property name="geometry">
                <rect>
                    <x>344</x>
                    <y>70</y>
                    <width>327</width>
                    <height>251</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="title">
                <string>Publication:</string>
            </property>
            <widget class="QWidget" name="layoutWidget_4">
                <property name="geometry">
                    <rect>
                        <x>12</x>
                        <y>41</y>
                        <width>291</width>
                        <height>191</height>
                    </rect>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <item>
                        <layout class="QVBoxLayout" name="verticalLayout_2">
                            <item>
                                <widget class="QLabel" name="label_10">
                                    <property name="maximumSize">
                                        <size>
                                            <width>100</width>
                                            <height>16777215</height>
                                        </size>
                                    </property>
                                    <property name="font">
                                        <font>
                                            <pointsize>10</pointsize>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string>Title:</string>
                                    </property>
                                    <property name="textInteractionFlags">
                                        <set>Qt::NoTextInteraction</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="label_11">
                                    <property name="maximumSize">
                                        <size>
                                            <width>100</width>
                                            <height>16777215</height>
                                        </size>
                                    </property>
                                    <property name="font">
                                        <font>
                                            <pointsize>10</pointsize>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string>ISBN13:</string>
                                    </property>
                                    <property name="textInteractionFlags">
                                        <set>Qt::NoTextInteraction</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="poliLabel_0">
                                    <property name="maximumSize">
                                        <size>
                                            <width>100</width>
                                            <height>16777215</height>
                                        </size>
                                    </property>
                                    <property name="font">
                                        <font>
                                            <pointsize>10</pointsize>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                    <property name="textInteractionFlags">
                                        <set>Qt::NoTextInteraction</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="poliLabel_2">
                                    <property name="maximumSize">
                                        <size>
                                            <width>100</width>
                                            <height>16777215</height>
                                        </size>
                                    </property>
                                    <property name="font">
                                        <font>
                                            <pointsize>10</pointsize>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                    <property name="textInteractionFlags">
                                        <set>Qt::NoTextInteraction</set>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QVBoxLayout" name="verticalLayout">
                            <item>
                                <widget class="QLabel" name="titleLabel">
                                    <property name="font">
                                        <font>
                                            <pointsize>10</pointsize>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                    <property name="textInteractionFlags">
                                        <set>Qt::NoTextInteraction</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="isbnLabel">
                                    <property name="font">
                                        <font>
                                            <pointsize>10</pointsize>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                    <property name="textInteractionFlags">
                                        <set>Qt::NoTextInteraction</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="poliLabel_1">
                                    <property name="font">
                                        <font>
                                            <pointsize>10</pointsize>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                    <property name="textInteractionFlags">
                                        <set>Qt::NoTextInteraction</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="poliLabel_3">
                                    <property name="font">
                                        <font>
                                            <pointsize>10</pointsize>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                    <property name="textInteractionFlags">
                                        <set>Qt::NoTextInteraction</set>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                </layout>
            </widget>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
